archive.md

```diff
@@ -5,390 +5,123 @@ title: Archiv
-    - [matplotlib](/files/archive/2023/matplotlib.html){:target="_blank"} (Stand 26.09.2023)
-    - [Aufgaben](/files/archive/2023/exercises-toolbox-2.zip) (Stand 26.09.2023)
+    - [Numeric Python](/files/archive/current/numeric-python.html){:target="_blank"} (Stand 26.09.2023)
+    - [matplotlib](/files/archive/current/matplotlib.html){:target="_blank"} (Stand 26.09.2023)
+    - [Aufgaben](/files/archive/current/exercises-toolbox-2.zip) (Stand 26.09.2023)
 - Tag 3
-    - [Scientific Python](/files/archive/2023/scientific-python.html){:target="_blank"} (Stand 27.09.2023)
-    - [uncertainties](/files/archive/2023/uncertainties.html){:target="_blank"} (Stand 27.09.2023)
-    - [Aufgaben](/files/archive/2023/exercises-toolbox-3.zip) (Stand 27.09.2023)
+    - [Scientific Python](/files/archive/current/scientific-python.html){:target="_blank"} (Stand 27.09.2023)
+    - [uncertainties](/files/archive/current/uncertainties.html){:target="_blank"} (Stand 27.09.2023)
+    - [Aufgaben](/files/archive/current/exercises-toolbox-3.zip) (Stand 27.09.2023)
 - Tag 4
-    - [Unix](/files/archive/2023/unix.pdf){:target="_blank"} (Stand 28.09.2023)
-    - [Make](/files/archive/2023/make.pdf){:target="_blank"} (Stand 28.09.2023)
-    - [Aufgaben](/files/archive/2023/exercises-toolbox-4.zip) (Stand 28.09.2023)
+    - [Unix](/files/archive/current/unix.pdf){:target="_blank"} (Stand 28.09.2023)
+    - [Make](/files/archive/current/make.pdf){:target="_blank"} (Stand 28.09.2023)
+    - [Aufgaben](/files/archive/current/exercises-toolbox-4.zip) (Stand 28.09.2023)
 - Tag 5
-    - [git](/files/archive/2023/git.pdf){:target="_blank"} (Stand 29.09.2023)
-    - [Aufgaben](/files/archive/2023/exercises-toolbox-5.zip) (Stand 29.09.2023)
+    - [git](/files/archive/current/git.pdf){:target="_blank"} (Stand 29.09.2023)
+    - [Aufgaben](/files/archive/current/exercises-toolbox-5.zip) (Stand 29.09.2023)
 
 ### LaTeX
-- [Folien](/files/archive/2023/latex.pdf){:target="_blank"} (Stand 06.10.2023)
-- [Aufgaben Tag 1](/files/archive/2023/exercises-latex-1.zip) (Stand 02.10.2023)
-- [Aufgaben Tag 2](/files/archive/2023/exercises-latex-2.zip) (Stand 04.10.2023)
-- [Aufgaben Tag 3](/files/archive/2023/exercises-latex-3.zip) (Stand 04.10.2023)
-- [Aufgaben Tag 4](/files/archive/2023/exercises-latex-4.zip) (Stand 05.10.2023)
+- [Folien](/files/archive/current/latex.pdf){:target="_blank"} (Stand 06.10.2023)
+- [Aufgaben Tag 1](/files/archive/current/exercises-latex-1.zip) (Stand 02.10.2023)
+- [Aufgaben Tag 2](/files/archive/current/exercises-latex-2.zip) (Stand 04.10.2023)
+- [Aufgaben Tag 3](/files/archive/current/exercises-latex-3.zip) (Stand 04.10.2023)
+- [Aufgaben Tag 4](/files/archive/current/exercises-latex-4.zip) (Stand 05.10.2023)
 - <a href="https://github.com/pep-dortmund/toolbox-workshop-protocol-template" target="_blank">LaTeX Vorlage für Protokolle</a>
 
+#### Alles zusammen als Archive
```

Comment:
Maybe change "Archive" to German singular "Archiv"? This sounds somewhat off otherwise